orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -169,6 +176,9 @@ def forward(ctx, *inputs, **kwargs):
                 def backward(ctx, *grad_outputs):
                     '''Performs backward pass based on grad wrt module output
                     '''
+                    # Use IO binding
+                    # Push user output grads to ONNX backend.
+                    backward_grad_output_ortvalue = []
```

Comment:
is this needed? it is initialized at line 199

Reply:
No. I don't intend to changed this. Removed